src/braket/default_simulator/noise_operations.py

```diff
@@ -45,6 +45,12 @@ def matrices(self) -> List[np.ndarray]:
     def targets(self) -> Tuple[int, ...]:
         return self._targets
 
+    @property
+    def probabilities(self):
+        return [
+            self._probability,
+        ]
```

Comment:
Not sure how I feel about `probabilities` for a noise operation with a single probability. Not necessarily opposed though

Reply:
It felt a little odd to me too. Any ideas on how we can do it better ?

Reply:
We could do a "probability" property for those, but I'm not sure which is better

Reply:
I was thinking that.

But while accessing it here: https://github.com/aws/amazon-braket-default-simulator-python/pull/167/files#diff-21541b28f861cc99c2edabe9f30038e3763e44e1d3841b83eb627738145f041fR91

I have to do two getattr

I thought this code smell is better than writing two getattr

Reply:
If you go with the single dispatch approach, then that won't be an issue. You could just have one implementation access `noise_op.probability` and one `noise_op.probabilities`. Still I'm unsure of which approach is better. I think maybe Cody would be a good person to weigh in once he's back

Reply:
For now, I will go with separating  noise_op.probability and noise_op.probabilities.

Will check with cody too, once he is back